test/test_main.cpp

```diff
@@ -1,73 +1,32 @@
 /**
  * @file
  * @brief     Test file
- * @author    Jeroen van Hattem
+ * @author    Jeroen van Hattem, Wiebe van Breukelen
  * @license   MIT
  */
 #define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
-#include "../src/coordinate3d.hpp"
+
 #include "../src/robot_arm.hpp"
```

Comment:
Can't you include these directly? So without ```..src/```?

Reply:
Done!